status-im/status-react
# feature #4065 - gas and gas price can be changed for dapp transactions

fixes #4065 

### Summary:

Adds the option to pass gas and gas limit for transactions created in dapps. The changes involve:

- `status-go` upgrade to `develop-g68966af0` for the dependency endpoints
- updated ios and android native modules to use the new endpoints
- updated UI to allow gas editing for dapp transactions

### Testing notes (optional):

Every send transaction option, not just from dapp (e.g. Wallet Send, chat command, both ether and tokens, etc.) - all where touched by the change.


### Steps to test:
- Send a transaction from dapp
- Tap Advanced, open gas settings and change them
- Send transaction and check the gas details in Transaction history and on Etherscan
- The same thing for other transaction flows

status: ready
